css/properties.schema.json

```diff
@@ -12,12 +12,53 @@
         }
       }
     },
-    "stringList": {
+    "groupList": {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
       "items": {
```

Comment:
Thanks for pointing that out! This got actually redundant, because we have the data in definitions.json. So I just removed this item in https://github.com/mdn/data/pull/99/commits/a2f237368dfd06d0b9501c825a61a442b98fd6ce.

Sebastian